src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -50,7 +50,8 @@ public void periodic() {
 
     public CommandBase spinUp() {
```

Comment:
Do we need to pull in some of @AustinFortunato's shooter code?

Reply:
Probably but when he re-bases with master it should